scio-extra/src/main/scala/com/spotify/scio/extra/breeze/breeze.scala

```diff
@@ -17,8 +17,8 @@
 
 package com.spotify.scio.extra
 
-import breeze.linalg.operators.OpAdd
-import breeze.linalg.support.CanCopy
+import _root_.breeze.linalg.operators.OpAdd
```

Comment:
Why do we need to prefix all these imports?

Reply:
Because I moved the existing Breeze stuff into its own `breeze` package, which collides with the root namespace.